.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "Appium/tests/helpers/mocks"]
 	path = Appium/tests/helpers/mocks
 	url = https://github.com/jasonkarns/bats-mock
+[submodule "Appium/tests/helpers/bats"]
```

Comment:
I depends how one views it. In this case I feel git submodules is a very valid use case and actually most of the examples and README itself uses Bats this way and by default the files in those submodules are not even inside the repo, repo just contains the info on them. But if @SrinivasanTarget thinks npm is better, I'm fine with that, 90% of tests execution will happen on travis either way most likely.